pkg/manifests/manifests.go

```diff
@@ -3228,6 +3269,8 @@ func (f *Factory) TelemeterClientDeployment(proxyCABundleCM *v1.ConfigMap) (*app
 			d.Spec.Template.Spec.Containers[i].Image = f.config.Images.PrometheusConfigReloader
 		case "kube-rbac-proxy":
 			d.Spec.Template.Spec.Containers[i].Image = f.config.Images.KubeRbacProxy
+		case "kube-rbac-proxy-metric":
```

Comment:
```suggestion
		case "kube-rbac-proxy-metrics":
```

Reply:
we reuse kube-rbac-proxy, i'm going to remove this case "kube-rbac-proxy-metrics".